docker/init.dev.sh

```diff
@@ -1,13 +1,5 @@
-echo "Install bundler stuff"
-bundle update --full-index --conservative exercism_config
-
-echo "Setup Exercism Config"
-bundle exec setup_exercism_config
-
 echo "Create and migrate DBs"
-bundle exec bin/rails db:create
-bundle exec bin/rails db:migrate
+bundle exec bin/rails db:create db:migrate
```

Comment:
Does this cause `migrate` to still run even if `create` exits with "db already exists"?

Reply:
Good question... LOL.  I can revert this is need be.